Realm/RLMMigration.h

```diff
@@ -87,6 +87,16 @@ typedef void (^RLMObjectMigrationBlock)(RLMObject *oldObject, RLMObject *newObje
  */
 - (void)deleteObject:(RLMObject *)object;
 
+/**
+ Deletes the data for the class with the given name.
+ This is used to clean up as removing RLMObject subclasses.
```

Comment:
This deletes all objects of the given class, and if the RLMObject subclass no longer exists in your program, cleans up any remaining metadata for the class in the Realm file.